ui/apps/platform/src/Containers/SystemConfig/Details/PublicConfigTelemetryDetails.tsx

```diff
@@ -36,7 +36,8 @@ const PublicConfigTelemetryDetails = ({
             <CardBody>
                 <p className="pf-u-mb-sm">
                     Online telemetry data collection allows Red Hat to use anonymized information to
-                    enhance your user experience.
+                    enhance your user experience. Consult the documentation to see what is collected,
+                    and for how to opt out.
```

Comment:
```suggestion
                    and for information about how to opt out.
```
(if there is enough space - if not, leave as is)